src/main.ts

```diff
@@ -160,15 +176,18 @@ async function run(): Promise<void> {
     );
 
     await cache.restore();
-    await runCargoSemverChecks(cargo);
+    const output = await runCargoSemverChecks(cargo);
+    core.setOutput("logs", stripAnsi(output));
```

Comment:
I just realized something. By making output available in the `logs` output value, are we accidentally preventing it from being printed into the GitHub workflow logs themselves?

That might be shooting ourselves in the foot, because we might prevent maintainers from seeing the output if they need to dig in for any reason. It'd be awesome to have the ability to post run output as a comment in the PR, but I don't think we want to break other users that use the action differently (e.g. before `cargo publish` like `cargo-semver-checks` uses it itself).

Reply:
> I just realized something. By making output available in the logs output value, are we accidentally preventing it from being printed into the GitHub workflow logs themselves?

What do you mean? In my tests I can see the logs in both GitHub Action run and in the comment.